src/translations/en.json

```diff
@@ -3689,9 +3689,12 @@
             "title": "Configure network interfaces",
             "connected_to": "Connected to {ssid}",
             "scan_ap": "Scan for access points",
+            "signal_strength": "Signal strength",
             "open": "Open",
             "wep": "WEP",
             "wpa": "wpa-psk",
+            "wifi": "Wi-Fi",
```

Comment:
Never paid much attention, but isn't the more common convention just "WiFi" (no hyphen)?

Reply:
I stand corrected.